docker-compose.yml

```diff
@@ -19,6 +19,17 @@ services:
     expose:
       - "9000"
 
+  redis:
+    image: redis:5 # current stable version, v6 out between March 2020 and May 2020
+    hostname: redis
+    entrypoint: /usr/local/bin/redis-server --appendonly yes --appendfsync everysec # using combination of RDB and AOF for persistence: https://redis.io/topics/persistence
+    volumes:
+      - redis-data:/data
+    networks:
+      - xain-fl-dev
```

Comment:
shouldn't this be `xain-fl`?

Reply:
Most definitely. That's called bad-copy-and-paste! :man_facepalming: